libraries/common/include/GazeboYarpPlugins/common.h

```diff
@@ -52,6 +66,17 @@ namespace GazeboYarpPlugins {
             return false;
         }
     }
+
+    inline double convertDegreesGainsToRadiansGains(double gainsWrtDegrees)
+    {
+        return gainsWrtDegrees * 180.0 / pi;
```

Comment:
Let's dothe math for proportional gains:
[gainsWrtDegrees] = [outputQuantity] / deg 
[gainsWrtRadians] = [outputQuantity] / rad 
so gainsWrtRadians = gainsWrtDegrees * deg/rad = gainsWrtDegrees * 180.0 / pi
But if you need to ask it clearly need a better explanation in the documentaiton.